doc/source/dev/contributor/cython.rst

```diff
@@ -57,7 +57,7 @@ performance. This ``.pyx`` file is automatically converted to a ``.c``
 file by Cython when SciPy is built.
 
 The only thing left is to add an extension from this ``.c`` file using
-|distutils|_. This takes just a single line in |optimize-setup|_:
+:doc:`numpy.distutils <numpy:reference/distutils>`. This takes just a single line in |optimize-setup|_:
```

Comment:
I am not necessarily proposing this change unless interlinks are preferred in all cases.  The issue here is that there doesn't appear to be any way to use an interlink in an alias, but removing the substitution makes the raw text less readable and presents some formatting differences to be addressed.